### PR TITLE
BETA: Register rikubot.is-a.dev

### PR DESCRIPTION
**Pull request by devops-riku**
Added `rikubot.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/rikubot.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devops-riku",
+        "email": "egermino.riku@gmail.com"
+    },
+    "record": {
+        "CNAME": "rikudev.cloud"
```
Comment (CuteDog5695) [inline domains/rikubot.json:7]: seems to just redirect to Hostinger page, can you add your website here?

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/rikubot.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/rikubot.json:7]: so you dont have a website hosted atm?

Comment (ThisCatLikesCrypto) [reply domains/rikubot.json:7]: you need _something_ to be able to register

Comment (devops-riku) [reply domains/rikubot.json:7]: > so you dont have a website hosted atm?

yeah, exactly

Comment (devops-riku) [reply domains/rikubot.json:7]: kindly check my site again, rikudev.cloud, I hosted my test page, thank you

Comment (ThisCatLikesCrypto) [reply domains/rikubot.json:7]: I don't have write access but there's something now

**Review by ThisCatLikesCrypto — APPROVED** [review]

**Review by devops-riku — COMMENTED** [review]
For Approval

File: domains/rikubot.json (hunk 1 above)
Comment (devops-riku) [reply domains/rikubot.json:7]: No worries, Ill wait for the approval thankss

**Review by iostpa — APPROVED** [review]
I think its fine now since site has something

**Pull request merged**